### PR TITLE
Release 0.3.0

### PR DESCRIPTION
**Pull request by shellforge**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,187 +1,38 @@
 # Warpgrapher
+[![Build Status](https://github.com/warpforge/warpgrapher/workflows/Test/badge.svg)](https://github.com/warpforge/warpgrapher/actions?query=workflow%3A%22Test%22+branch%3Amaster)
+[![GitHub Release](https://img.shields.io/github/release/warpforge/warpgrapher.svg?style=flat)](https://github.com/warpforge/warpgrapher/releases/tag/v0.2.0)
```
Comment (maldrake) [inline README.md]: Does this tag need to be updated to 0.3.0?

File: README.md [hunk 2]
```diff
@@ -1,187 +1,38 @@
...
-models. Describe the data model for which you want to run a web service.
-Wargrapher automatically generates a GraphQL schema from the data model, as well
-as a set of resolvers for basic reate, read, update, and delete (CRUD)
-operations on that data.
+#### Framework for creating data model-driven GraphQL API services
 
-If you need more more sophisticated, custom queries and endpoints, you can
-supply your own custom resolvers. Warpgrapher will automatically generate the
-GraphQL configuration and invoke your custom resolvers when appropriate.
+As a developer you can focus on defining your applications data model and warpgrapher takes care of generating a graph-based API for interacting with that model. In addition to generating CRUD APIs for interacting with the model, warpgrapher provides a set of advanced features to customize and extend your service.
```
Comment (maldrake) [inline README.md]: The fragment "your applications," should be "your application's."  Also, "data model and warpgrapher" should be "data model, and warpgrapher."  Should warpgrapher be capitalized when used, even mid-sentence?  Is is a proper noun, as a named project, so I would think yes, but as long as we're consistent, I'm okay with either.

File: book/src/engine/endpoints_defined.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Defined Endpoints
 
-In addition to the CRUD endpoints auto-generated for each type, a Warpgrapher app serve additional defined GraphQL endpoints (root queries/mutations).
+In addition to the CRUD endpoints auto-generated for each type, Warpgrapher provides the ability to define additional endpoints.
```
Comment (maldrake) [inline book/src/engine/endpoints_defined.md:3]: Either other places that say "warpgrapher," should be capitalized, or this mid-sentence use should be lower-case, to be consistent.

**Review by maldrake — COMMENTED** [review]
The examples are all neo4j.  I didn't see conditional compilation feature flags in the source. How do you keep them from blowing up if you do a `cargo build --features cosmos`?

File: README.md (hunk 1 above)
Comment (shellforge) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (shellforge) [reply README.md]: Fixed (using uppercase).

File: book/src/engine/endpoints_defined.md (hunk 3 above)
Comment (shellforge) [reply book/src/engine/endpoints_defined.md:3]: Fixed elsewhere (using uppercase).

**Comment by shellforge** [comment]
@maldrake  

> The examples are all neo4j. I didn't see conditional compilation feature flags in the source. How do you keep them from blowing up if you do a `cargo build --features cosmos`?

Confirmed that `cargo build` does not build examples. I added one more step to the pipeline that will check that the examples compile.

**Review by maldrake — APPROVED** [review]

**Pull request merged**